flytekit/annotated/launch_plan.py

```diff
@@ -228,3 +229,36 @@ def get_registerable_entity(self) -> SdkLaunchPlan:
             version=settings.version,
         )
         return self._registerable_entity
+
+
+class ReferenceLaunchPlan(ReferenceEntity, LaunchPlan):
+    def __init__(
+        self, project: str, domain: str, name: str, version: str, inputs: Dict[str, Type], outputs: Dict[str, Type]
+    ):
+        super().__init__(_identifier_model.ResourceType.LAUNCH_PLAN, project, domain, name, version, inputs, outputs)
+
+    def get_registerable_entity(self) -> SdkLaunchPlan:
```

Comment:
another instance where I would love to move the Code from here into a separate package

Reply:
yes, agreed.